src/librbd/cache/WriteLogCache.h

```diff
@@ -18,10 +19,10 @@ template <typename> class ImageCacheState;
 }
 
 template <typename ImageCtxT>
-class WriteLogCache : public ImageCache<ImageCtxT> {
+class WriteLogCache {
```

Comment:
Nit: I am thinking that since this no longer is tied to the `ImageCache`, perhaps this should really be the `WriteLogImageDispatch` hook. I think @MahatiC's plan was to use `WriteLogCache` to satisfy the interface of `ImageCache` and then have it call down into either the RWL or SSD write-log implementation.

Reply:
Let me discuss with @MahatiC , and update it in the next PR if needed. Is it OK?

Reply:
Yeah, that was the intention of `WriteLogCache`. I think `WriteLogCache` can now be the `WriteLogImageDispatch` hook and RWL/SSD can be invoked with `AbstractWriteLog m_image_cache;` in the `WriteLogImageDispatch`. Will discuss this with Lisa.